client/app/views/dashboard/repos/repos.html

```diff
@@ -45,9 +45,11 @@
 
           <span ng-hide="repo.build" class="name">{{ repo.label }}</span>
 
-      <span ng-show="repo.build" class="name">
-        <a ui-sref="dashboard.ciBuild({ buildId: parseInt(repo.build.id) })">{{ repo.label }}</a>
-      </span>
+          <span ng-show="repo.build" class="name">
+            <a ui-sref="dashboard.ciBuild({ buildId: parseInt(repo.build.id) })">{{ repo.label }}</a>
+          </span>
+
+          <span class="font-error" ng-show="!repo.build.enabled && repo.build.disable_reason == 'no_config_file'"> No .shoov.yml file </span>
```

Comment:
@amitaibu Probably design should be changed (other message, should disappear after couple of seconds or should contain link to tutorial etc).